packages/sdk/src/controllers/LayoutController.ts

```diff
@@ -339,4 +339,35 @@ export class LayoutController {
         const res = await this.#editorAPI;
         return res.getLayoutPrivateData(id).then((result) => getEditorResponseData<PrivateData>(result));
     };
+
+    /**
+     * Sets the display name for any layout
+     * @param id the id of the layout to update
+     * @param displayName the display name
+     * @returns
+     */
+    setDisplayName = async (id: string, displayName: string | null) => {
```

Comment:
They probably should use the `reset` version instead?

```suggestion
    setDisplayName = async (id: string, displayName: string) => {
```